tex/generic/pgf/frontendlayer/tikz/libraries/tikzlibrarymindmap.code.tex

```diff
@@ -110,17 +101,41 @@
 }}%
 \tikzset{every circle connection bar/.style={}}%
 
+\pgfkeys{/pgf/decoration/start flare length/.initial=0pt}%
+\pgfkeys{/pgf/decoration/end flare length/.initial=0pt}%
+
 \def\tikz@compute@circle@radii{%
   \pgf@process{\pgfpointtransformed{\pgfpointanchor{\tikztostart}{center}}}%
   \pgf@xa=\pgf@x%
   \pgf@process{\pgfpointtransformed{\pgfpointanchor{\tikztostart}{west}}}%
   \advance\pgf@xa by-\pgf@x%
   \pgfkeys{/pgf/decoration/start radius/.expanded=\the\pgf@xa}%
+  \pgfutil@tempdima=.5\pgf@xa\relax%
   \pgf@process{\pgfpointtransformed{\pgfpointanchor{\tikztotarget}{center}}}%
   \pgf@xa=\pgf@x%
   \pgf@process{\pgfpointtransformed{\pgfpointanchor{\tikztotarget}{west}}}%
   \advance\pgf@xa by-\pgf@x%
   \pgfkeys{/pgf/decoration/end radius/.expanded=\the\pgf@xa}%
+  \advance\pgfutil@tempdima by.5\pgf@xa\relax%
```

Comment:
OK, but this makes me doubly interested in the answer to my question above about whether newdimen is problematic, since it seems to be used so infrequently.

Reply:
OK, I found a better way.